tests/e2e/specs/in-game/one-offs/9_nines.spec.js

```diff
@@ -200,7 +195,146 @@ describe('Playing NINES', () => {
+      cy.get('[data-opponent-face-card=11-1]').click();
+
+      cy.resolveOpponent();
+
+      assertGameState(0, {
+        p0Hand: [],
+        p0Points: [ Card.TEN_OF_SPADES, Card.ACE_OF_SPADES ],
+        p0FaceCards: [],
+        p1Hand: [ Card.JACK_OF_DIAMONDS, Card.NINE_OF_HEARTS ],
+        p1Points: [],
+        p1FaceCards: [],
+      });
+
+      // Opponent nines Jack
+      cy.playTargetedOneOffOpponent(Card.NINE_OF_HEARTS, Card.JACK_OF_SPADES, 'jack');
+      cy.get('[data-cy=cannot-counter-resolve]').click();
+  
+      assertGameState(0, {
+        p0Hand: [ Card.JACK_OF_SPADES ],
+        p0Points: [ Card.TEN_OF_SPADES, ],
+        p0FaceCards: [],
+        p1Hand: [ Card.JACK_OF_DIAMONDS ],
+        p1Points: [ Card.ACE_OF_SPADES ],
+        p1FaceCards: [],
+      });
+    });
+
+    it('Plays a 9 on a triple jack to scrap all jack', () => {
+      cy.loadGameFixture(0, {
+        p0Hand: [ Card.ACE_OF_SPADES, Card.NINE_OF_CLUBS, Card.JACK_OF_SPADES ],
+        p0Points: [ Card.TEN_OF_SPADES ],
+        p0FaceCards: [],
+        p1Hand: [ Card.JACK_OF_CLUBS, Card.JACK_OF_DIAMONDS, Card.NINE_OF_HEARTS ],
+        p1Points: [],
+        p1FaceCards: [],
+      });
+
+      // Player plays Ace of Spades
+      cy.get('[data-player-hand-card=1-3]').click();
+      cy.get('[data-move-choice=points]').click();
+
+      assertGameState(0, {
+        p0Hand: [ Card.NINE_OF_CLUBS, Card.JACK_OF_SPADES ],
+        p0Points: [ Card.TEN_OF_SPADES, Card.ACE_OF_SPADES ],
+        p0FaceCards: [],
+        p1Hand: [ Card.JACK_OF_CLUBS, Card.JACK_OF_DIAMONDS, Card.NINE_OF_HEARTS ],
+        p1Points: [],
+        p1FaceCards: [],
+      });
+
+      // Opponent plays jack
+      cy.playJackOpponent(Card.JACK_OF_CLUBS, Card.ACE_OF_SPADES);
+
+      assertGameState(0, {
+        p0Hand: [ Card.NINE_OF_CLUBS, Card.JACK_OF_SPADES ],
+        p0Points: [ Card.TEN_OF_SPADES ],
+        p0FaceCards: [],
+        p1Hand: [ Card.JACK_OF_DIAMONDS, Card.NINE_OF_HEARTS ],
+        p1Points: [ Card.ACE_OF_SPADES ],
+        p1FaceCards: [],
+      });
+
+
+      // player plays jack
+      cy.get('[data-player-hand-card=11-3]').click();
+      cy.get('[data-move-choice=jack]').click();
+      cy.get('#player-hand-targeting').should('be.visible');
+      cy.get('[data-opponent-point-card=1-3]').click({ force: true });
+
+      // Opponent plays jack
+      cy.playJackOpponent(Card.JACK_OF_DIAMONDS, Card.ACE_OF_SPADES);
+
+      // player plays 9
+      cy.get('[data-player-hand-card=9-0]').click();
+      cy.get('[data-move-choice=targetedOneOff]').click();
+      cy.get('#player-hand-targeting').should('be.visible');
+      cy.get('[data-opponent-point-card=1-3]').click({ force: true });
+
+      cy.resolveOpponent();
+
+      assertGameState(0, {
+        p0Hand: [  ],
+        p0Points: [ Card.TEN_OF_SPADES ],
+        p0FaceCards: [],
+        p1Hand: [ Card.NINE_OF_HEARTS, Card.ACE_OF_SPADES ],
+        p1Points: [],
+        p1FaceCards: [],
```

Comment:
Shall we assert the scrap here to confirm the jacks were scrapped along with the nine?